src/collective/exportimport/import_content.py

```diff
@@ -399,6 +409,140 @@ def import_new_content(self, data):  # noqa: C901
 
         return added
 
+    def import_versions(self, container, item):
+        """Import one item with all its revisions..
+
+        TODO: Decide if we want to apply hooksfor each version or only for the current object.
+        """
+        portal_workflow = api.portal.get_tool("portal_workflow")
+
+        # Disable automatic versioning!
+        portal_types = api.portal.get_tool("portal_types")
+        fti = portal_types.get(item["@type"])
+        repo_tool = api.portal.get_tool("portal_repository")
+        policy = None
+        policies = repo_tool._version_policy_mapping.get(item["@type"], [])
+        versioning_behavior = None
```

Comment:
This code part could do with a small explanation: "We are temporarily disabling versioning and will re-enable it at the end.".